### DISCUSSION TITLE
加载问题

### DISCUSSION THREAD

**Discussion by jackTJT** [Tencent/libpag#2091] [General] [answered]
在进行PAGCompostion后，替换图片或者文本后，libpag会重新加载pag文件还是说直接从缓存加载

**Comment by kevingpqi123** [comment, marked as answer]
基于 PAGFile 或 PAGComposition 替换图片或编辑文本，PAGFile 或 PAGComposition 的对象又没有发生变化，不还是同一个指针吗
